source/get-started/bare-metal-install-server.rst

```diff
@@ -62,15 +62,42 @@ Follow these steps to install |CL| on the target system:
 
 #. Reboot the target system.
 
-#. This action launches the |CL| installer boot menu, shown in Figure 1.
+Choose boot menu option
+=======================
+
+#. Choose one of the options shown in Figure 1.
+
+   a. Follow `Verify integrity of installer media (optional)`_.
+
+   #. Select :guilabel:`Clear Linux OS` in the boot menu.
 
    .. figure:: /_figures/bare-metal-install-server/bare-metal-install-server-01.png
       :scale: 100%
       :alt: Clear Linux OS Installer boot menu
 
       Figure 1: Clear Linux OS Installer boot menu
 
-#. With :guilabel:`Clear Linux OS` highlighted, select :kbd:`Enter`.
+   .. note::
+
+      If no action is taken, installation starts by default.
```

Comment:
Technically, the installation will not start, it just boots into the live image.

```suggestion
         If no action is taken, the live image starts by default.
```

Reply:
@mdhorn Thanks for the attention to wording. I applied your suggestions. See latest commit.